kivy/uix/bubble.py

```diff
@@ -303,7 +301,7 @@ def on_arrow_pos(self, *l):
         self_arrow_layout.clear_widgets()
         self_arrow_img = self._arrow_img
         self._sctr = self._arrow_img
-        self.clear_widgets(do_super=True)
+        self._super_clear_widgets()
```

Comment:
```suggestion
        super(Bubble, self).clear_widgets()
```
Calling super directly should work.